README.md

```diff
@@ -16,26 +17,29 @@ Redirecting URLs is an often discussed topic, especially when it comes to
 SEO. Previously it was hard to create a "real" redirect without performance
 loss or incorrect handling. But this time is over!
 
-With the Redirect Module setting up redirects will become easier than ever before!
-
 ## Setup
-- Add `@nuxtjs/redirect-module` dependency using yarn or npm to your project
-- Add `@nuxtjs/redirect-module` to `modules` section of `nuxt.config.js`
-- Configure it:
 
-### Inline options
+1. Add `@nuxtjs/redirect-module` dependency with `yarn` or `npm` into your project
+2. Add `@nuxtjs/redirect-module` to `modules` section of `nuxt.config.js`:
+3. Configure it:
+
 ```js
 {
   modules: [
-    ['@nuxtjs/redirect-module', [ /* Redirect option here */]],
+    // Simple usage
```

Comment:
Why do we have that "Simple usage" comment everywhere? :thinking:

Reply:
Removed, no need :+1: